SIPS/sip-29.md

```diff
@@ -225,10 +229,10 @@ type FungibleAssetUnit = {
 // Fungible asset metadata.
 type FungibleAssetMetadata = {
   // Human-friendly name of the asset.
-  name: string;
+  name?: string;
```

Comment:
Does the asset controllers and UI support this? If not, we may need to keep this as is